tests/testsuite/nix_shell.rs

```diff
@@ -28,7 +28,7 @@ fn pure_shell() -> io::Result<()> {
         .output()?;
     let actual = String::from_utf8(output.stdout).unwrap();
 
-    let expected = format!("via {} ", Color::Red.bold().paint("pure"));
+    let expected = format!("via {} ", Color::Blue.bold().paint(" pure"));
```

Comment:
Is this U+F313? It falls into the *private use area*, meaning that every font is free to display any glyph at all for this code point. It's usually not a good idea to use private use area Unicode code points unless your application controls the font.

Reply:
Good point. 👍 
It may be best to use `❄️` here, and add the NerdFont symbol to the NerdFont preset: 
https://github.com/starship/starship/blame/master/docs/presets/README.md#L17